### ISSUE TITLE
Add support of "python -m pip" 

### ISSUE THREAD

**Issue by extsoft** [hadolint/hadolint#454]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

There is [DL3013](https://github.com/hadolint/hadolint/wiki/DL3013) that check whether versions are pinned or not when new Python packages being installed. And it works - means `hadolint` founds a violation` if you have a line like this one

```dockerfile
FROM python:3.8-alpine
RUN pip install flask
```

However, there is another way how you can use `pip`: `python -m pip` instead of `pip`. And in a Dockerfile, it may look as follows

```dockerfile
FROM python:3.8-alpine
RUN python -m pip install flask
```

I would expect the same output for both Dockerfiles.

### Actual behavior

Now, `hadolint` does not show a violation for `python -m pip`.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

I've used <https://hadolint.github.io/hadolint/> for testing; however, it fails in the command line also.

```bash
~ docker run --rm hadolint/hadolint hadolint --version
Haskell Dockerfile Linter v1.18.0-6-ga0d655d
```

**Comment by davidhart82** [comment]
There is a test that `python3 -m pip install test==1` passes; but there is not a test that `python3 -m pip install test` doesn't pas.
